### PR TITLE
Added buildDeck()

### PR DESCRIPTION
**Pull request by IStallcup**
Sorry not in a branch... :-1:  But! Created buildDeck() function, started to populate it. Used integers to place-hold for the not-yet-created Card class.

### CODE REVIEW

**Comment by IStallcup** [comment]
References issue #4

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -8,6 +8,33 @@
 * Created by IStallcup on 2/27/16.
 */
 
+
+//abstract class
 public class Game {
-	//May change, present for initial commit of Game class
+	
+	//create the deck list
+	public.java.util.List</*card type goes here*/int> deck = new ArrayList<>();
```
Comment (leel8on) [inline src/main/java/models/Game.java]: int needs to be changed to Integer. It needs an object.

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -8,6 +8,33 @@
...
+	//use int as placeholder as the suit of the card doesn't matter in blackjack
+
+
+	//create the heand for each player?
+	public.java.util.List</*card type goes here*/int> hand = new ArrayList<>();
```
Comment (leel8on) [inline src/main/java/models/Game.java]: int needs to be changed to Integer. It needs an object.

**Comment by leel8on** [comment]
no branch? am disappoint

Anyway, make the changes and I think the build should pass, and then I would be willing to merge.

**Comment by athai** [comment]
*cough*Add a test*cough*

**Comment by IStallcup** [comment]
Made those changes! Test passes and it only took me like 5 minutes to figure out StupidJ.

**Comment by athai** [comment]
Lgtm +1

**Pull request merged**